capstone/src/test/javascript/graphsSpec.js

```diff
@@ -0,0 +1,56 @@
+const google = {};
+google.visualization = {};
+google.visualization.DataTable = class {
+  /**
+   * Dummy constructor for mock DataTable class.
+   */
+  constructor() {}
+
+  /**
+   * Dummy function for adding rows to mock DataTable class.
+   *
+   * @param {Array} values the array of pairs corresponding to a row.
+   */
+  addRow(values) {}
+
+  /**
+   * Dummy function for adding columns to mock DataTable class.
+   *
+   * @param {String} type the datatype of the column.
+   * @param {String} name the name of the column.
+   */
+  addColumn(type, name) {}
+};
+
+
+describe('Data Table Construction', () => {
```

Comment:
I just notice this is part of a file named graphSpec. Should we add it to a test file? For example graph_test.js

Reply:
The naming convention for the test files with the Jasmine framework is to use `Spec` at the end, otherwise I would have named it with `test` or something similar.

Reply:
I see, sounds good.